enterprise/frontend/src/metabase-enterprise/license/components/BillingInfo/BillingInfoError.tsx

```diff
@@ -16,6 +16,7 @@ export const BillingInfoError = () => {
             <strong>{t`Need help?`}</strong>{" "}
             {t`You can ask for billing help at `}
             <strong>
+              {/* eslint-disable-next-line i18next/no-literal-string */}
               <Anchor href="mailto:billing@metabase.com">
```

Comment:
I wonder if we can configure `i18next/no-literal-string` to ignore href attributes, as we would never want it to be localized